layout/_layout.swig

```diff
@@ -93,5 +93,25 @@
   {% include '_third-party/scroll-cookie.swig' %}
   {% include '_third-party/exturl.swig' %}
   {% include '_third-party/bookmark.swig' %}
+
+    <script type="text/javascript">
+        var OriginTitile=document.title;
```

Comment:
personalized too much...😅

Reply:
@ivan-nginx Can we consider adding this feature (change to a different title when window is no longer in focus) to the list #67 ?

Reply:
@ApolloZhu @ivan-nginx https://github.com/theme-next/hexo-next-title